libminifi/include/utils/StringUtils.h

```diff
@@ -105,6 +105,10 @@ class StringUtils {
     return s;
   }
 
+  static std::string_view trim(const std::string_view& sv);
+
+  static std::string_view trim(const char* str);
```

Comment:
These two could be replaced with a single `trim(std::string_view input)`.  Also, by looking around the internets, I got the impression that passing `string_view` by value is generally preferable to passing it by (const) reference.

Reply:
you are right about the const ref, changed it to by value

unfortunately we cannot remove the `const char*` overload, as if we want to keep the `std::string` overload, it wouldn't be able to disambiguate which conversion (`const char* -> std::string_view` vs `const char* -> std::string`) to apply

removing the `std::string` overload would work, but it seems to me that we can freely convert a temporary `std::string` to `std::string_view` resulting in accidental dangling references